src/clarity-angular/layout/tabs/tab-link.directive.ts

```diff
@@ -38,7 +39,7 @@ export class TabLinkDirective {
 
     constructor(public ifActiveService: IfActiveService, @Inject(IF_ACTIVE_ID) private id: number,
                 private ariaService: AriaService, private el: ElementRef, private cfr: ComponentFactoryResolver,
-                private viewContainerRef: ViewContainerRef) {
+                private viewContainerRef: ViewContainerRef, private tabs: Tabs) {
```

Comment:
This may seem like a nitpick but I would like you to make the change: to avoid the whole circular dependency with `forwardRef()` and all, you should just declare a provider for the tabs id that's a simple value, and inject that instead of `Tabs`. It will also let you run the unit tests for tab links without having to wrap them in a tabs component, which is much better isolation.

Reply:
@youdz I completely overlooked an option of using providers. I wanted to actually use a service first, but I thought that would just make it complicated. I will opt for providers. Thanks for the suggestion